content/blog/2016/2016-04-20-pipeline-2.x.adoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Pipeline 2.x plugins
+tags:
+- pipeline
+- jenkins2
+author: jglick
+---
+
+Those of you who routinely apply all plugin updates may already have noticed that the version numbers of the plugins in the Pipeline suite have switched to a 2.x scheme. Besides aligning better with the upcoming Jenkins 2.0 core release, the plugins are now being released with independent lifecycles.
+
+“Pipeline 1.15” (the last in the 1.x line) included simultaneous releases of a dozen or so plugins with the 1.15 version number (and 1.15+ dependencies on each other). All these plugins were built out of a single `workflow-plugin` repository. While that was convenient in the early days for prototyping wide-ranging changes, it has become an encumbrance now that the Pipeline code is fairly mature, and more people are experimenting with additions and patches.
+
+As of 2.0, all the plugins in the system live in their own repositories on GitHub—named to match the plugin code name, which in most cases uses the historical _workflow_ term, so for example `workflow-job-plugin`. Some complex steps were moved into their own plugins, such as `pipeline-build-step-plugin`. The 1.x changelog is closed; now each plugin keeps a changelog in its own wiki, for example https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Job+Plugin[here] for the Pipeline Job plugin.
```

Comment:
I wish to have Documentation publishing on Jenkins CI ASAP. surfing across multiple repos is complicated for users